### ISSUE TITLE
vXbox/PYXInput support

### ISSUE THREAD

**Issue by rodjun** [rodjun/general_selfdriving_ai#1]
TLDR; This will decrease the number of requirements and make setting up an environment much easier, BUT it requires loading an unsigned driver so safeboot must be disabled.

So we have two options: 

Hard setup: Needs a library that is not on pip, requires installing vjoy and x360ce in the games folder with a certain specific configuration (shouldn't be too much of a problem for people if i write down every step properly)

Easy setup: Everything can be installed from pip, only requirement outside of pip which is installing the driver is already described on PYXInputs readme, but requires system-level configuration changes (disabling safeboot)

**Comment by rodjun** [comment]
Fixed in https://github.com/rodjun/general_selfdriving_ai/commit/4591b57ca15ec93b5b03d33629f206c3ec8bae69
